api/src/main/java/org/eclipse/microprofile/openapi/models/security/SecurityRequirement.java

```diff
@@ -34,8 +34,8 @@
      * required scope (optional) provided.
      * 
      * @param securitySchemeName the name of security scheme declared in the Components section of the OpenAPI document
-     * @param scope a required scope - only valid when the defined scheme is 'oauth2' or 'openIdConnect'. null values will be rejected
-     *              (implementation will throw an exception) or ignored.
+     * @param scope a required scope - only valid when the defined scheme is 'oauth2' or 'openIdConnect'. passing null will
```

Comment:
since we're updating this line, I would change `when the defined scheme is` -> `when the defined scheme's type is`